### PR TITLE
Icecream parlor search

### PR DESCRIPTION
**Pull request by hlhartley**
Adds a drop down menu to search by parlor.
If user types in ice cream flavor and searches by parlor, it will only display the ice cream flavors in that parlor.

### CODE REVIEW

**Comment by ericweissman** [comment]
Not sure why this was closed. I'm going to leave a few more comments but I like everything so far!

File: src/App.js [hunk 1]
```diff
@@ -43,15 +45,24 @@ class App extends Component {
...
+            searchParlor={this.searchParlor} 
+            parlors={this.state.parlors}
+          />
           <Banner />
           <CardContainer parlors={this.state.parlors} 
                         icecream={this.state.icecream}
-                        search={this.state.value}/>
+                        search={this.state.value}
+                        searchByParlor={this.state.searchByParlor}/>
           {/* <ErrorMessage /> */}
```
[the remark this answers is not in the document]
Comment (hlhartley) [reply src/App.js]: This component was recently built after getting some feedback from test users. We can decide if we'd like to implement it or if we like the single search function better

File: src/SearchByParlor.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import './css/Main.scss';
+import { arch } from 'os';
+
+const SearchByParlor = (props) => {
+    let parlors = props.parlors
+    return (
+        <div className="SearchByParlor">
+            <select onChange={props.searchParlor}>
+                <option value="" selected disabled hidden>Select Parlor</option>
+                {parlors.map((parlor) => {
```
[the remark this answers is not in the document]
Comment (hlhartley) [reply src/SearchByParlor.js]: That's a great idea! It doesn't sound like it'd be too challenging to do

File: src/Banner.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const Banner = () => {
           {/* <h2>Interesting ideas that set your mind in motion</h2> */}
         <div className='banner-center'>
           <img src='images/icecreams.jpg' className='cones-img'/>
-          <p>Hear directly from the people who know it best. From tech to politics to creativity and more — whatever your interest, we’ve got you covered.</p>
+          <p className='banner-text'>Our mission is to show you the craziest, interesting, new ice cream flavors on the block around Denver and Aurora. You can use the search bar to locate your favorite ice cream or just browse our homepage to see the coolest new flavors!</p>
```
[the remark this answers is not in the document]
Comment (hlhartley) [reply src/Banner.js]: Sure. I'll change this

File: src/css/Card.scss [hunk 4]
```diff
@@ -40,18 +40,18 @@
...
-    margin-top: 230px;
+    margin-top: 25px;
 }
 
 .parlor-popup {
     display: flex;
     position: fixed;
     z-index: 2;
-    justify-content: center;
+    // justify-content: center;
```
[the remark this answers is not in the document]
Comment (hlhartley) [reply src/css/Card.scss]: Yes, we have a lot of unused CSS and commented out comments. I'll start refactoring the SASS / variables first

File: src/css/ParlorDetail.scss [hunk 5]
```diff
@@ -7,34 +7,45 @@
   left: 0;
   right: 0;
   bottom: 0;
-  background-color: #99ccff;
+  background-color: #fff8de;
+  border: 2px solid black;
```
[the remark this answers is not in the document]
Comment (TomWilhoit) [reply src/css/ParlorDetail.scss]: Paletton.com is really good about generating complimentary color pairings with existing palettes.

File: src/css/Card.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (TomWilhoit) [reply src/css/Card.scss]: A lot of use cases here for those variables in SCSS.

**Comment by TomWilhoit** [comment]
This was a great feature, Heather! I'll be letting some people try it out over the next day to see if there is any more user feedback we can get.

**Pull request merged**